src/txn/txn_rollback_to_stable.c

```diff
@@ -109,13 +109,13 @@ __rollback_abort_update(WT_SESSION_IMPL *session, WT_ITEM *key, WT_UPDATE *first
     }
 
     /*
-     * Clear the history store flag for the stable update to indicate that this update should not be
-     * written into the history store later, when all the aborted updates are removed from the
+     * Clear the history store flags for the stable update to indicate that this update should not
```

Comment:
`should be` and `not`?

Reply:
I think the correct grammar is "should not be"

Reply:
Sorry for not providing the clarity, the sentence I feel is `should be`, there is no need for `not` as it may be misleading, because we are clearly the flag to ensure that this update is inserted again into the history store.

Reply:
right

Reply:
[d2a9a54](https://github.com/wiredtiger/wiredtiger/pull/8131/commits/d2a9a54d33d6a7945d4132d9be159a218c19b509)